@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -269,15 +270,29 @@ export class VisibleTileSet {
 
         const newRenderList: DataSourceTileList[] = [];
         let allVisibleTilesLoaded: boolean = true;
+        let allBoundingBoxesFinal: boolean = true;
 
         if (this.options.extendedFrustumCulling) {
             this.m_mapTileCuller.setup();
         }
 
+        let elevationRangeSource: ElevationRangeSource | undefined;
+        for (const dataSource of dataSources) {
+            elevationRangeSource = dataSource.getElevationRangeSource();
+            if (elevationRangeSource !== undefined) {
```

Comment:
I notice you check for the tiling scheme below, does it make more sense to check here? Else you could get an ElevationRangeSource which won't match.

Reply:
Assuming we have multiple datasources some with webMercator tiling scheme and some with equirectangular tiling scheme. The current behaviour will be that all webMercator data sources will use proper bounding boxes and all equirectangular data sources will have flat bounding boxes.
You mean to change the behavior so that also the webMercator data sources get flat bounding boxes if there is any equirectangular(or whatever) data source